pkg/master/master.go

```diff
@@ -774,7 +782,85 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 	return version
 }
 
-func (m *Master) InstallThirdPartyAPI(rsrc *experimental.ThirdPartyResource) error {
+func (m *Master) HasThirdPartyResource(rsrc *expapi.ThirdPartyResource) (bool, error) {
```

Comment:
Done, though what is our policy on Godoc vis a vis interface implementations.  Shouldn't we defer to the comment in the interface specification?